SUPPORTED_MODELS.md

```diff
@@ -623,7 +623,7 @@
 	<tr align="center">
 		<td>2</td>
 		<td><b>LabelBinarizer</b></td>
-        <td>>=0.1</td>
+        <td>>=0.8</td>
```

Comment:
We did support transporting the `LabelBinarizer` preprocessing model from version 0.1, it was crucial for handling Linear models.